dbt-cta/impactive/models/1_cta_full_refresh/_1_cta_full_refresh__models.yml

```diff
@@ -0,0 +1,1237 @@
+        description: ''
+      - name: voterbase_id
+        description: ''
+      - name: reported_by_email
+        description: ''
+      - name: tag_name
+        description: ''
+      - name: tag_slug
+        description: ''
+      - name: created_at
+        description: ''
+      - name: exported_at
+        description: ''
+      - name: reportable_type
+        description: ''
+      - name: van_id
+        description: ''
+      - name: updated_at
+        description: ''
+      - name: phone
+        description: ''
+      - name: user_id
+        description: ''
+      - name: tag_id
+        description: ''
+      - name: id
+        description: ''
+        tests:
+          - not_null
+          - unique
+      - name: fullname
+        description: ''
+      - name: taggable_id
+        description: ''
+      - name: reported_by_fullname
+        description: ''
+      - name: value
+        description: ''
+      - name: email
+        description: ''
+      - name: campaign_id
+        description: ''
+      - name: taggable_type
+        description: ''
+      - name: _airbyte_ab_id
+        description: ''
+        tests:
+          - not_null
+          - unique
+      - name: _airbyte_emitted_at
+        description: ''
+        tests:
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: timestamp
+      - name: _airbyte_normalized_at
+        description: ''
+        tests:
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: timestamp
+      - name: _airbyte_taggings_export_hashid
+        description: ''
+
+  - name: teams_export_base
+    description: ''
+    columns:
+      - name: member_phone
+        description: ''
+      - name: member_email
+        description: ''
+      - name: member_user_id
+        description: ''
+      - name: member_performs
+        description: ''
+      - name: team_id
+        description: ''
+      - name: team_rank
+        description: ''
+      - name: team_leaderboard_score
+        description: ''
+      - name: team_name
+        description: ''
+      - name: exported_at
+        description: ''
+      - name: team_leaderboard_performs
+        description: ''
+      - name: member_fullname
+        description: ''
+      - name: team_leader_ids
+        description: ''
+      - name: campaign_id
+        description: ''
+      - name: member_score
+        description: ''
+      - name: _airbyte_ab_id
+        description: ''
+        tests:
+          - not_null
+          - unique
+      - name: _airbyte_emitted_at
+        description: ''
+        tests:
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: timestamp
+      - name: _airbyte_normalized_at
+        description: ''
+        tests:
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: timestamp
+      - name: _airbyte_teams_export_hashid
```

Comment:
I think hash_id would be a better uniqueness check than airbyte_ab_id if theres no primary key column like id in the other tables

Reply:
Yeah, there was no obvious pk i could see in this table. Adding the uniqueness check to the hash_id sounds good. Flagging that the `airbyte_ab_id` tests are automagically added from the dbt test script. Would love to chat more with the team, mayhaps, about how we can make the dbt tests a bit tighter?

Reply:
@jitoquinto - i added a uniqueness test here. would love another peek/approval when you get a chance.

Reply:
yay thank you! And yeah agreed, it would be good to take another look at the script and re-evaluate default ones. I personally don't think were getting much value out of checks on the `_airbyte_ab_id` tests but would be good to discuss! maybe a good eng weekly topic for when @emily-flambe can attend

and yep taking a look :cat_typing: